### ISSUE TITLE
Recognize touchMove events that started outside of the target element

### ISSUE THREAD

**Issue by AdrianLoer** [exyte/Macaw#285]
*I am rolling my own scrollView with a Macaw view at its center and do not yet understand everything about events in swift. It's possible I broke something somewhere UIView related:*

When I set a onTouchesMoved handler on multiple node elements only the one on the first node touched will be triggered.

Is there a way to get this right so that I can get a continuous stream of nodes that are under the finger's location?

Or am I doing something wrong and that should work out of the box? I used the PeriodicTable example and could not get it to work there either

**Comment by ystrot** [comment]
Hey Adrian,

Yes, that's how move events work. When you touch one node and move your finger, all move events will be delivered to this first node. If you want to detect node under the finger then you need to add listener on MacawView and found which node selected manually.

**Comment by AdrianLoer** [comment]
Thank you for the quick response, great work you are doing here

Use case is that I want to highlight different nodes as I drag along. I now added the following to the MacawView

````
public func findNodeAt(loc: CGPoint) -> Node? {
        
        guard let renderer = renderer else {
            return nil
        }
        var foundNode: Node? = .none
        localContext { ctx in
            foundNode = renderer.findNodeAt(location: loc, ctx: ctx)
        }
        return foundNode
    }
````

This is not perfect and mostly works because I have an exported SVG from Sketch with proper IDs to discard unwanted events on nodes but for me right now its enough. Touch locations are captured on the paren UIView anyways.

But due to the private nature of involved properties I had to modify MacawView at source level.
If there is a better way any information is appreciated.

Or maybe a general hit test function could be published similar to mine above?

**Comment by ystrot** [comment]
Yep, good point. We need to add such a public function to `MacawView`.
